components/img/welcome/welcomebg.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please run this svg through https://jakearchibald.github.io/svgomg/

Reply:
let's do the png opt in a follow up